_data/hosting_providers.json

```
@@ -2298,5 +2298,15 @@
     "plan": "",
     "reviewed": "2020.01.29",
     "note": "Shared Web Hosting utilizes Plesk Panel interface to enable Let's Encrypt; Managed Cloud Hosting enabled via Certbot on request."
+  },
+    {
+    "name": "CA Net, Europe",
+    "link": "https://www.canet.at",
+    "category": "full",
+    "tutorial": "",
+    "announcement": "https://www.canet.at/hosting-server/",
+    "plan": "All plans",
+    "reviewed": "2020.02.06",
+    "note": "All vServers and Webspace with Froxlor or I-MSCP use Let's Encrypt for Web and Mail"
```

I suggest we generalize this to make it clear it refers to all plans:

```suggestion
    "note": "All plans come with Froxlor or I-MSCP and use Let's Encrypt for Web and Mail"
```